### DISCUSSION TITLE
Java library a Kiegészítő lapok adataihoz

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#111] [Kérdések és válaszok - Gépi interfész]
Kedves NAV Szakértők,

a kiegészítő lapok adatai jellemzően nem kerülnek tárolásra a vállalatirányítási rendszerekben. Viszont ahhoz, hogy ezekkel együtt lehessen beküldeni az eÁFA adatokat, szükséges őket valamilyen módon felrögzíteni/ feltölteni oda.

A kérdésem ez lenne, hogy ennek az elősegítésére tervben van-e olyan Java library publikálása, ami tartalmazza a kiegészítő lapok adatstruktúráját?

A választ előre is köszönöm.

**Comment by u319169** [comment]
Kedves Szalaierika!
A NAV jelenleg nem készít külső fejlesztők által használható Java könyvtárakat. Ezeknek megvalósítását egyelőre mindenkinek saját kezűleg kell elkészíteni.
Üdvözlettel: u319169
